pkg/webhook/router.go

```diff
@@ -16,12 +15,6 @@ import (
 )
 
 func Register(manager controllerruntime.Manager, webhookList ...Webhook) error {
-	// skipping webhook setup if certificate is missing
-	certData, _ := ioutil.ReadFile("/tmp/k8s-webhook-server/serving-certs/tls.crt")
-	if len(certData) == 0 {
-		return nil
-	}
-
```

Comment:
If I understood correctly, `controller-manager` will start automatically the webhook server if we got certificates in the right path, isn't it?

Reply:
yep, we don't need it because we reconcile certificates before starting the webhook server. And if the reconciliation won't be successful we just do `exit(1)`